pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/loader/IndexLoadingConfig.java

```diff
@@ -56,6 +56,7 @@ public class IndexLoadingConfig {
   private final InstanceDataManagerConfig _instanceDataManagerConfig;
   private final TableConfig _tableConfig;
   private final Schema _schema;
+  private boolean _enableSegmentPreprocess = true;
```

Comment:
Suggest renaming it to `_skipSegmentPreprocess` so that the default is `false`. Adding default true flag is error prone.